x-pack/plugins/profiling/public/components/no_data_page.tsx

```diff
@@ -251,6 +251,49 @@ docker.elastic.co/observability/profiling-agent:stable /root/pf-host-agent \\
         },
       ],
     },
+    {
+      key: 'symbols',
+      title: i18n.translate('xpack.profiling.tabs.symbols.title', {
+        defaultMessage: 'Upload Symbols',
+      }),
+      steps: [
+        {
+          title: i18n.translate('xpack.profiling.tabs.symbols.step1', {
+            defaultMessage: 'Download the symbtool binary from the URL below:',
+          }),
+          content: (
+            <EuiLink target="_blank" href={`https://ela.st/symbtool-linux-amd64`}>
+              https://ela.st/symbtool-linux-amd64
```

Comment:
It looks like this URL points to the same place as https://ela.st/elastic-profiling-linux-amd64 (see https://links.elastic.dev/alias/search?term=symbtool-linux-amd64).

So at this time the user would download the `elastic-profiling` executable instead. I don't recall whether the official executable for symbol uploads is `elastic-profiling` or `symbtool`.

Reply:
Yes.